### PR TITLE
[Toolchain] Remove default toolchain info message.

### PR DESCRIPTION
**Pull request by jyoungyun**
This commit removes default toolchain info message. This feature is not stable, showing toolchain message confuses user even more.

ONE-vscode-DCO-1.0-Signed-off-by: Jiyoung Yun <jy910.yun@samsung.com>

### CODE REVIEW

File: src/Toolchain/DefaultToolchain.ts [hunk 1]
```diff
@@ -32,49 +32,12 @@ class DefaultToolchain {
...
-        });
-    });
-  }
-
   public set(toolchainEnv: ToolchainEnv, toolchain: Toolchain) {
     if (this.isEqual(toolchain)) {
       return;
     }
     this._toolchainEnv = toolchainEnv;
     this._toolchain = toolchain;
```
Comment (dayo09) [inline src/Toolchain/DefaultToolchain.ts:40]: If you're gonna remove the information message, how about replacing it with balloon or logger(console)?

Comment (jyoungyun) [reply src/Toolchain/DefaultToolchain.ts:41]: Updated this patch. Please see the latest code.

**Review by dayo09 — APPROVED** [review]
Thanks!

**Review by llFreetimell — APPROVED** [review]
LGTM

**Pull request merged**